source/leadership.rst

```diff
@@ -0,0 +1,133 @@
+yt-project from a high level. They are expected to coordinate their actions
+with other red members so that the project presents a coherent position towards
+the outside. They are expected to prepare and attend the monthly team calls and
+executive committee meetings. We expect red members to teach and grow other
+members so that they can pass their organizational responsibilities on to them.
+This encourages us to increase diversity, reduce burn-out and allow group
+members to shift their focus in reaction to changing life circumstances. If a
+red member chooses to change focus, that change of focus will be announced at
+the next monthly team meeting. Red group members are expected to also perform
+duties of blue group members while facilitating and helping the blue group to
+grow.
+
+**Blue Group** (includes all subproject deputies and yt project members) – This
+is where the journey of a new yt project team member starts. Blue group members
+spend a large part of your time facilitating contributions from others to the
+project (e.g. reviewing work, answering questions, maintaining project
+infrastructure, doing triage of new issues). We expect blue members to teach
+and grow contributors that can become blue members. This encourages diversity
+of the project. After building a good understanding of the technology and
+social structure of the project blue members can change to a red member by
+volunteering for tasks which a red member is expected to perform. After a while
+of doing so the group will ask if you want to become a red member and, if you
+agree, recognise you as such at a monthly team meeting.
+
+**Green Group** (can be any person, any time, self-nominated only)-- This group
+applies to members of the community whose life situation has changed so that
+they prefer to (temporarily) not take on the rights and responsibilities of
+being an active member. Membership to the green group is invoked by
+self-nomination. They can return to active membership by resuming their
+community activities. Their return will be announced at the next team meeting
+or via the developers mailing list.
+
+**Emeritus Group** (time based or self-nominated)-- this group is for members
+of the community at any level that have a prolonged leave of member duties from
+the yt project. We welcome participation in the community for any length of
+time, and this is a way to retire from the project. A project member may become
+emeritus by self-nomination or by not interacting in a membership capacity on
+the yt project for over 2 years. After this time, a member of the executive
+committee will contact an inactive member and see if they wish to remain active
+or transition to an emeritus role. 
+
+Finally, members of the community that wish to not be associated with the yt
+project may request to be removed from any/all teams/groups/committees and
+documentation by e-mailing either confidential@yt-project.org or
```

Comment:
A note to come back to later, this email address is dead.

Reply:
Wow, that's not great. We should definitely make this not dead. Madicken would you be OK with setting up an e-mail account to replace this one? It doesn't need to be on the yt-project.org domain if that would be a pain. Maybe @Xarthisius knows more and can fix that e-mail address?

I think it might also be worthwhile to make this e-mail account go to someone who is not normally a yt developer and to publicize that fact along with the address so that if someone has a problem with someone in the yt leadership they feel like their report will be black holed by the person they are reporting.